canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.scss

```diff
@@ -82,19 +73,59 @@ $primary-tab-height: $spacing-08;
 	}
 }
 
-.properties-category-content {
-	height: 100%;
-	padding: $spacing-05;
-	border-bottom: 1px solid $layer-accent-01;
-	display: none;
-	&.show {
-		display: inherit;
+.properties-subtabs,
+.properties-primaryTabs {
+	.cds--tabs__nav {
+		overflow-x: auto; // override carbon so scrollbar doesn't always show
 	}
 }
 
-.properties-subtabs, .properties-primaryTabs {
-	.cds--tabs__nav {
-		overflow-x: auto; // override carbon so scrollbar doesn't always show
+.properties-categories {
+	.cds--accordion__item--active .cds--accordion__content {
+		// 24px padding between accordion title and content (issue #3106)
+		padding: $spacing-02 $spacing-05 $spacing-05;
```

Comment:
Remove the comment on line 85.
This padding doesn't look correct. In test harness, there's `padding: $spacing-05`  added -
![Screenshot 2024-04-18 at 12 51 58 PM](https://github.com/elyra-ai/canvas/assets/25124000/3812294a-10b6-4e4b-b2d1-65cdcf334176)